app/Models/GameSet.php

```diff
@@ -53,12 +75,40 @@ public function games(): BelongsToMany
     /**
      * @return BelongsToMany<GameSet>
      */
-    public function links(): BelongsToMany
+    public function parents(): BelongsToMany
+    {
+        return $this->belongsToMany(GameSet::class, 'game_set_links', 'child_game_set_id', 'parent_game_set_id')
+            ->withTimestamps()
+            ->withPivot('created_at', 'updated_at');
+    }
+
+    /**
+     * @return BelongsToMany<GameSet>
+     */
+    public function children(): BelongsToMany
     {
         return $this->belongsToMany(GameSet::class, 'game_set_links', 'parent_game_set_id', 'child_game_set_id')
             ->withTimestamps()
             ->withPivot('created_at', 'updated_at');
     }
 
     // == scopes
+
+    /**
+     * @param Builder<GameSet> $query
+     * @return Builder<GameSet>
+     */
+    public function scopeCentralHub(Builder $query): Builder
+    {
+        $centralHubId = Cache::rememberForever('central_hub_id', function () {
```

Comment:
I'm not fond of this solution. Can't we just force insert a `Central` hub after truncating the table, then go back to hardcoding the value (to 1)?

Reply:
Could even consider force inserting the hubs that appear in the menu so they'll work for the seeded DB too (how does seeded data interact with the sync command)?

Reply:
I like that idea.

In latest, I've added this logic to the sync command which inserts the 5 "standard" hubs with hardcoded IDs of 1-5.

If the target games "backing" the hubs don't exist (ie: not a prod snapshot), the hubs are created anyway.

Realistically, someone on a seeded database will likely never execute this command. Therefore, given that these standard hubs are nav items, it probably also makes sense to seed these standard hubs.

A tiny new seeder has been added in latest:
```php
public function run(): void
{
    foreach (self::STANDARD_HUBS as $title => $id) {
        GameSet::unguard(); // temporarily allow filling the "id" field
        GameSet::create([
            'id' => $id,
            'title' => $title,
            'type' => GameSetType::Hub,
        ]);
        GameSet::reguard();
    }
}
```